tests/lib/fakestore/cmd/fakestore/cmd_make_refreshable.go

```diff
@@ -24,12 +24,16 @@ import (
 )
 
 type cmdMakeRefreshable struct {
-	TopDir string `long:"dir" description:"Directory to be used by the store to keep and serve snaps, <dir>/asserts is used for assertions"`
+	TopDir     string `long:"dir" description:"Directory to be used by the store to keep and serve snaps, <dir>/asserts is used for assertions"`
+	SnapBlob   string `long:"snap-blob"`
```

Comment:
It's a bit confusing how snap-blob is different from snap name, can you add a description?